worker/uniter/actions/resolver.go

```diff
@@ -42,26 +42,47 @@ func (r *actionsResolver) NextOp(
 	opFactory operation.Factory,
 ) (operation.Operation, error) {
 	nextAction, err := nextAction(remoteState.Actions, localState.CompletedActions)
-	if err != nil {
+	if err != nil && err != resolver.ErrNoOperation {
```

Comment:
If there are no operation left to be run, then we cannot return the error signaling such here, we must first check to see if an action is already running (that has been interrupted) before we declare that there is nothing to do (by returning a `resolver.ErrNoOperation` error)